### PR TITLE
Code Cleanup

### PR DESCRIPTION
**Pull request by Cinnazeyy**
Fix issue: https://github.com/AlpsBTE/Plot-System/issues/112

### CODE REVIEW

File: src/main/java/com/alpsbte/plotsystem/commands/CMD_Tpll.java [hunk 1]
```diff
@@ -47,98 +47,98 @@
...
+
+        Player player = (Player) sender;
+        World playerWorld = player.getWorld();
+
+        if (PlotManager.isPlotWorld(playerWorld)) {
+            player.sendMessage(Utils.getErrorMessageFormat(LangUtil.get(sender, LangPaths.Message.Error.CANNOT_TELEPORT_OUTSIDE_PLOT)));
+            return true;
+        }
+
+        String[] splitCoords = args[0].split(",");
```
Comment (LordTuxn) [inline src/main/java/com/alpsbte/plotsystem/commands/CMD_Tpll.java:74]: args can be null

File: src/main/java/com/alpsbte/plotsystem/commands/review/CMD_Review.java
Comment (LordTuxn) [inline src/main/java/com/alpsbte/plotsystem/commands/review/CMD_Review.java:1]: needs to be tested

File: src/main/java/com/alpsbte/plotsystem/core/menus/ReviewMenu.java
Comment (LordTuxn) [inline src/main/java/com/alpsbte/plotsystem/core/menus/ReviewMenu.java:1]: needs to be tested

File: src/main/java/com/alpsbte/plotsystem/core/menus/PlotMemberMenu.java
Comment (LordTuxn) [inline src/main/java/com/alpsbte/plotsystem/core/menus/PlotMemberMenu.java:1]: needs to be tested

File: src/main/java/com/alpsbte/plotsystem/core/menus/ReviewPlotMenu.java
Comment (LordTuxn) [inline src/main/java/com/alpsbte/plotsystem/core/menus/ReviewPlotMenu.java:1]: needs to be tested

**Review by LordTuxn — CHANGES_REQUESTED** [review]
I tested the plugin a bit and some menus/commands no longer work properly. All changes need to be tested on the server before merging.

File: src/main/java/com/alpsbte/plotsystem/commands/admin/CMD_DeletePlot.java [hunk 2]
```diff
@@ -29,39 +29,33 @@
...
-                    }
-                } else {
-                    sender.sendMessage(Utils.getErrorMessageFormat("Could not find plot with ID #" + plotID + "!"));
-                }
-            } else {
-               sendInfo(sender);
-            }
-        } else {
+        if(!sender.hasPermission(getPermission())) {
             sender.sendMessage(Utils.getErrorMessageFormat("You don't have permission to use this command!"));
```
[the remark this answers is not in the document]
Comment (Cinnazeyy) [reply src/main/java/com/alpsbte/plotsystem/commands/admin/CMD_DeletePlot.java:39]: There wasn't any translation to begin with and i only fixed formatting! Also, to my knowledge this command has been on the "maybe-to-be-deleted-list" anyways, so i'm not sure if changing it is really necessary.

File: src/main/java/com/alpsbte/plotsystem/commands/admin/CMD_SetHologram.java [hunk 3]
```diff
@@ -39,45 +39,48 @@ public class CMD_SetHologram extends BaseCommand {
...
+            return true;
+        }
 
-                        // Update hologram location
-                        if(hologram != null) {
-                            hologram.setLocation(player.getLocation());
-                            player.sendMessage(Utils.getInfoMessageFormat("Successfully updated hologram location!"));
-                            player.playSound(player.getLocation(), Utils.Done,1,1);
+        if (!sender.hasPermission(getPermission())){
+            sender.sendMessage(Utils.getErrorMessageFormat("You don't have permission to use this command!"));
```
[the remark this answers is not in the document]
Comment (Cinnazeyy) [reply src/main/java/com/alpsbte/plotsystem/commands/admin/CMD_SetHologram.java:48]: same here, the goal of this PR is only to cleanup the whole code structure, not to change any logic

File: src/main/java/com/alpsbte/plotsystem/commands/CMD_Tpll.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Cinnazeyy) [reply src/main/java/com/alpsbte/plotsystem/commands/CMD_Tpll.java]: fixed in latest commit!

**Pull request merged**